go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ErmakovDmitriy/ctimefmt
```

Comment:
Please change to `github.com/Mottl/ctimefmt` since you are committing to the upstream repo.